containers/src/Kokkos_DualView.hpp

```diff
@@ -832,16 +836,17 @@ void
 deep_copy (DualView<DT,DL,DD,DM> dst, // trust me, this must not be a reference
            const DualView<ST,SL,SD,SM>& src )
 {
-  if(src.modified_flags.data()==NULL || dst.modified_flags.data()==NULL) {
+  if(src.modified_flags_data()==NULL || dst.modified_flags_data()==NULL) {
```

Comment:
Why do we need this? If the flags are null the deep_copy will not copy anything.

Reply:
Removed this part after talking with @crtrott about it